package.json

```diff
@@ -20,8 +20,10 @@
   "author": "Simple Icons Collaborators",
   "license": "CC0-1.0",
   "scripts": {
-    "build": "node scripts/build.js",
-    "build:testpage": "node scripts/buildTestpage.js",
+    "build": "npm run build:font && npm run build:testpage",
```

Comment:
I would say this should be `npm run build:font && npm run build:css`, or we should have a "build:package" script that does that. This is especially useful when the CI is going to release a new version, where we don't actually need to build the test page.

Which do you prefer @metaa?